src/PmEmail.js

```diff
@@ -0,0 +1,391 @@
+module.exports = {
```

Comment:
Hi @Tony-Ndichu. It will be good to have an email templates folder that will hold all email templates.

Reply:
I agree @PriscillaSam , I've hence refactored the codebase to include the changes.